lib/skylight/probes/elasticsearch.rb

```diff
@@ -0,0 +1,27 @@
+module Skylight
+  module Probes
+    module Elasticsearch
+      class Probe
+        def install
+          ::Elasticsearch::Transport::Transport::Base.class_eval do
+            alias perform_request_without_sk perform_request
+            def perform_request(method, path, *args, &block)
+              ActiveSupport::Notifications.instrument "request.elasticsearch",
```

Comment:
Currently this implementation is pretty simple, however, after using this in production for awhile I think we'll want to use more complex functionality. So I think a normalizer is a more future-proof approach